### DISCUSSION TITLE
No sync

### DISCUSSION THREAD

**Discussion by nailnn** [Chia-Network/chia-blockchain#10550] [Support]
Hello. The .CHIA database was transferred from disk C to disk F. Synchronization was completely completed and everything fell down.
Now the wallet is synchronized and the synchronization of the software itself does not work. Nodes appear as in the first screenshot. Then they disappear as in the second screenshot. At the very bottom where the connection address blinks one address but nothing happens, the Block section is constantly loading. The latest version. Fail config tried to delete and run again. Nothing changed. Help! 
1 - https://drive.google.com/file/d/1hhbp2yx0liEWdJUS_4vv3ed_lWvySKyr/view?usp=sharing
2- https://drive.google.com/file/d/1LtHkIiWAI_zA8olmwklT__NyI8-7gffM/view?usp=sharing
3 - https://drive.google.com/file/d/1PyEM-cxbJBnWnRgpySOzu2nCTryre0II/view?usp=sharing

**Comment by loppefaaret** [comment]
you are likely suffering from the issue discussed here: #9677
